Alignment/OfflineValidation/python/TkAlAllInOneTool/configTemplates.py

```diff
@@ -17,6 +17,16 @@
 
 ######################################################################
 ######################################################################
+loadGlobalTagTemplate="""
+#Global tag
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag,".oO[GlobalTag]Oo.")
```

Comment:
@hroskes this will work in general, but can you clarify what you mean by "specify other conditions through this interface" ? The recommended way to customize the Event Setup is via the toGet method of GlobalTag (cf. https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideFrontierConditions#Customization_of_Global_Tags)